### PR TITLE
Delete tf.reduce_mean on scalar

### PR DESCRIPTION
**Pull request by amchercashin**
Remove `tf.reduce_mean()` from `tf.reduce_mean(tf.nn.l2_loss())` in _tensorflow/tools/docker/notebooks/2_getting_started.ipynb_
According to [reference](https://www.tensorflow.org/versions/r0.7/api_docs/python/nn.html#l2_loss), `tf.nn.l2_loss()` always returns a scalar, thus no
need to average it.
At last code block where loss function is re-implemented by hand,there is
no _mean()_ operation.
Excuse me if something is wrong, have little to no exp. in python and tensorflow.

### CODE REVIEW

**Comment by girving** [comment]
Looks good.  Please rebase and squash to avoid all the merge commits.  We don't like them in PRs.  Once you do that, I can test and merge.

**Comment by vrv** [comment]
@tensorflow-jenkins: test this please

(all the extra commits are merges).

**Comment by amchercashin** [comment]
@girving: it was [says in another commit](https://github.com/tensorflow/tensorflow/pull/1639) that 

> merge commits don't show up in our history

Do i still need to rebase and squash? If i do, am I right that i can rebase in my local git, update this branch on github, and commits will be squashed in this pull requests automatically? Or I need to make a new branch and pull request with squashed commits?

**Comment by vrv** [comment]
No need to rebase / squash :)

@tensorflow-jenkins: test this please

**Comment by amchercashin** [comment]
thx :)
